gufe/protocols/protocoldag.py

```diff
@@ -78,21 +109,38 @@ class ProtocolDAGResult(GufeTokenizable, DAGMixin):
     result_graph : nx.DiGraph
         Graph of `ProtocolUnitResult`s as nodes, with directed edges to each
         `ProtocolUnitResult`'s dependencies.
+    transformation_key : Union[GufeKey, None]
+        Key of the `Transformation` that this `ProtocolDAGResult` corresponds
+        to, if applicable. This functions as a label for identifying the source
+        of this `ProtocolDAGResult`.
+    extends_key : Optional[GufeKey]
+        Key of the `ProtocolDAGResult` that this `ProtocolDAGResult` extends from.
+        This functions as a label for identifying the source of this `ProtocolDAGResult`;
+        it can be used to reconstruct the tree of extensions from a collection
+        of `ProtocolUnitResult`s.
```

Comment:
```suggestion
        of `ProtocolUnitResult`\s.
```

This is RST, not Markdown. (Which also probably means that none of this will render how you expect, but this change will at least get docs to build without the warning.)

Reply:
Oh thanks, I didn't realize this was a thing. Does this apply to any character following the backtick (e.g., punctuation)?

Reply:
Oh yeah, for RST we typically need double backticks, yes?